Rocket/Rocket/RocketsList/RocketsListView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  RocketsListView.swift
+//  Rocket
+//
+//  Created by Adela Mišicáková on 11.07.2022.
+//
+
+import SwiftUI
+import ComposableArchitecture
+import ComposablePresentation
+
+//MARK: - State
+
+struct AppState: Equatable {
+    var detailState: DetailState? = nil
+    var fetchingState = FetchingState.na
+    var rockets = [Rocket]()
+    var details = IdentifiedArrayOf<DetailState>()
+    var alert: AlertState<AppAction>?
+    
+    enum FetchingState: Equatable {
+        case na
+        case loading
+        case success([Rocket])
```

Comment:
Keď sa vrátim k tomu invalid state, tak už tu by to mohlo byť 
case success(IdentifiedArrayOf<DetailState>)
potom sa zbavíš aj property detials aj propery rockets.

Je to trochu náročnejšie na predstavu asi tak sa kľudne pýtaj keby niečo nie je jasné.